web/css/github/markdown.css

```diff
@@ -1,124 +1,83 @@
 /* from https://github.com/sindresorhus/github-markdown-css/blob/gh-pages/github-markdown.css */
 
-.markdown-body {
-/*REMOVED  -ms-text-size-adjust: 100%;

```

Comment:
I updated `markdown.css`, but I noticed that previous code was sometimes commented with `/*REMOVED`.
Should I also comment the code (when possible)?

Reply:
I would say yes, to preserve the customizations done by @Seldaek when starting to use that stylesheet

Reply:
I commented the code when required.

Reply:
I will resolve this conversation, please let me know if you need any other change.